src/assets/Standards/ERC20Standard.test.ts

```diff
@@ -10,18 +11,61 @@ const ERC20_MATIC_ADDRESS = '0x7d1afa7b718fb893db30a3abc0cfc608aacfebb0';
 describe('ERC20Standard', () => {
   let erc20Standard: ERC20Standard;
   let web3: any;
+  nock.disableNetConnect();
```

Comment:
Put this in a `beforeAll`?

Reply:
accidentally force-pushed instead of making a new commit, but its addressed here now: https://github.com/MetaMask/controllers/pull/667/commits/3e48605dcb64c65ac23b6d21d10f06cbf2cb6d1a